common/web/gesture-recognizer/src/engine/headless/gestures/matchers/gestureMatcher.ts

```diff
@@ -126,9 +126,8 @@ export class GestureMatcher<Type, StateToken = any> implements PredecessorMatch<
         baseContact = srcContact.baseSource;
       }
 
-      if(baseContact.isPathComplete) {
-        throw new Error("GestureMatcher may not be built against already-completed contact points");
-      }
```

Comment:
To be clear, this was never in place due to errors that occurred elsewhere; this was enforcing an early assumption about how the system could work.  I've since realized that said assumption is safe to relax.